packages/epinio/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,26 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,5 +1,14 @@
+ annotations:
+   artifacthub.io/license: Apache-2.0
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/experimental: "true"
++  catalog.cattle.io/kube-version: '>= 1.20'
++  catalog.cattle.io/namespace: cattle-epinio-system
++  catalog.cattle.io/os: linux
++  catalog.cattle.io/permits-os: linux,windows
++  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.100-0'
++  catalog.cattle.io/release-name: epinio
++  catalog.cattle.io/type: cluster-tool
```

Comment:
Please add the following:
1.  Add `catalog.cattle.io/display-name: 'Epinio'` for the name you want to be displayed for the chart in the UI (Doesn't need to be Epinio, that's just an example. Just make sure the string you set it to is capitalized).
2. Add `catalog.cattle.io/auto-install: epinio-crd=match` to let Rancher know this chart has a CRD chart associated with it and install it automatically.
3. Add a max Kubernetes version to `catalog.cattle.io/kube-version: '>= 1.20'`. If possible, adding a max is preferable to help prevent ambiguity when it comes to supported Kubernetes versions (`'>= 1.20 <= [max-k8s-version]'`). We are actively working on making this change to all of our other charts for the same reason.

Reply:
Addressed in commit 28ac1de26ee991527d098a8ca802b80716a1d274.
This is also rebased, with conflict resolved.
Regenerated chart in commit 1a900943cc06655946a22e5a3b518961ba20700a.